### PR TITLE
Updates to the FE to fix misc issues

### PR DESCRIPTION
**Pull request by manatarms**
These changes look scary but not a lot is changing. I did some house keeping.
The best way to review these changes are by commit. The last one is a formatting commit and can be ignored.

1 . Update webpack to accept OUTPUT_PATH variable                   0d3201c
2. Update login text to include API Tokens                                       24125eb
3. Update prettier config and dependency                                       869502f 
4. Fix 404s on upgrade links                                                             93ebdad
5. Update all paths to dashboard to not include legacy /a routes   28a1a61 
6.  Remove UTM and channel generation for links                           6b3bd72
7. Format code with prettier (Can be ignored)                                  a6c2022

### CODE REVIEW

**Review by mgirouard — APPROVED** [review]
lgtm

**Review by sejoker — APPROVED** [review]

File: src/components/AnalyticCard/AnalyticCard.js [hunk 1]
```diff
@@ -102,24 +104,26 @@ class AnalyticCard extends Component {
...
-                drawers={[
-                  {
-                    id: 'help',
-                    name: formatMessage({ id: 'container.drawer.help' }),
-                    content: <FormattedMarkdown text={this.props.helpTextId} />
-                  }
-                ]}
-              />
-            : null}
+          {this.props.helpTextId ? (
```
Comment (Xuran-1) [inline src/components/AnalyticCard/AnalyticCard.js:114]: `{this.props.helpTextId && <CardDrawers...` would be better

File: src/containers/App/App.js [hunk 2]
```diff
@@ -36,21 +35,25 @@ class AppContainer extends Component {
               className="content-wrapper"
               style={isLoggedIn() ? null : { marginTop: '-105px' }}
             >
-              {isLoggedIn()
-                ? <LayoutRow>
-                    <LayoutColumn width={1 / 1}><AppNavigation /></LayoutColumn>
-                  </LayoutRow>
-                : null}
+              {isLoggedIn() ? (
```
Comment (Xuran-1) [inline src/containers/App/App.js:38]: Same here: ` {isLoggedIn() && < LayoutRow>`

**Review by Xuran-1 — APPROVED** [review]

File: src/components/AnalyticCard/AnalyticCard.js (hunk 1 above)
Comment (manatarms) [reply src/components/AnalyticCard/AnalyticCard.js:114]: These are all formatting changes, so trying not to change anything unrelated to my updates.

Comment (Xuran-1) [reply src/components/AnalyticCard/AnalyticCard.js:114]: Sounds good!

**Review by milliechan — APPROVED** [review]

File: src/constants/Plans.js [hunk 3]
```diff
@@ -17,6 +17,21 @@ export function planNeedsUpgrade(currentPlan, minimumPlan) {
...
+  switch (minimumPlan) {
+    case FREE_PLAN:
+      return 'f';
+    case PRO_PLAN:
+      return 'p';
+    case BIZ_PLAN:
+      return 'b';
+    default:
+      // By default return a pro plan
+      // Note that we dont support
```
[the remark this answers is not in the document]
Comment (manatarms) [reply src/constants/Plans.js:30]: Good catch, my eyes probably glazed over at this point

File: src/containers/BypassCacheByCookieCard/BypassCacheByCookieCard.js [hunk 4]
```diff
@@ -36,18 +33,12 @@ class BypassCacheByCookieCard extends Component {
   render() {
     const { formatMessage } = this.props.intl;
 
-    let { activeZone, config, zones } = this.props;
+    let { activeZone, zones } = this.props;
```
[the remark this answers is not in the document]
Comment (manatarms) [reply src/containers/BypassCacheByCookieCard/BypassCacheByCookieCard.js:36]: Let me check

File: src/constants/Plans.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (manatarms) [reply src/constants/Plans.js]: Updated [here](https://github.com/cloudflare/cloudflare-plugin-frontend/pull/143/commits/e4ff9a5cffdc4db267948d4b9d86d3ed68137c6e)

File: src/containers/BypassCacheByCookieCard/BypassCacheByCookieCard.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (manatarms) [reply src/containers/BypassCacheByCookieCard/BypassCacheByCookieCard.js]: Updated [here](https://github.com/cloudflare/cloudflare-plugin-frontend/pull/143/commits/e4ff9a5cffdc4db267948d4b9d86d3ed68137c6e)

**Pull request merged**